Model/Server.py

```diff
@@ -1,28 +1,44 @@
-from sqlalchemy import Column, Integer, Text, create_engine
+from sqlalchemy import Column, Integer, String, DateTime, Boolean, create_engine
 from sqlalchemy.ext.declarative import declarative_base
+import datetime as dt
+from datetime import datetime
 
 Base = declarative_base()
 
 
-class Server(Base):
+class Instances(Base):
     """ This Class Store Detail of the Server"""
-    __tablename__ = "Server"
+    __tablename__ = "Instances"
     id = Column(Integer, autoincrement=True, primary_key=True)
-    ip = Column(Text, nullable=False)
-    url = Column(Text, nullable=False)
-    dbUser = Column(Text, nullable=False)
-    dbName = Column(Text, nullable=False)
-    dbPass = Column(Text, nullable=False)
-    sshUser = Column(Text, nullable=False)
-    sshPass = Column(Text, nullable=False)
+    ip = Column(String(100), nullable=False)
+    url = Column(String(100), nullable=False)
+    dbUser = Column(String(100), nullable=False)
+    dbName = Column(String(100), nullable=False)
+    dbPass = Column(String(100), nullable=False)
+    sshUser = Column(String(100), nullable=False)
+    sshPass = Column(String(100), nullable=False)
```

Comment:
Could you please define reason for using String with Length 100 rather than Text as Column Type.